onnx_tf/handlers/backend/conv_mixin.py

```diff
@@ -91,16 +91,47 @@ def conv(cls, node, input_dict, transpose=False):
                                         "Tensorflow")
 
     group = node.attrs.get("group", 1)
+    weight_shape = weights.get_shape().as_list()
+    # Is this convolution depthwise?
+    depthwise = (len(x_shape) == 4 and group == x_shape[1] and group != 1 and
+                 not transpose and len(weight_shape) == 4)
 
-    weight_groups = tf.split(weights, num_or_size_splits=group, axis=-1)
+    if depthwise is True:
+      # Depthwise convolution.
+      # The convolution kernel layout in tf.depthwise_conv is:
+      # [filter_height, filter_width, in_channels, channel_multiplier]
+      # Weight is now (KH x KW X C/g X M), or more precisely, (KH x KW X C/g X (g * M/g)),
+      # we reshape it to (KH x KW x C x M/g)
+      # NOTE: Assuming weight has fixed shape.
+      if weight_shape is None or None in weight_shape or group is None:
+        depthwise = False
```

Comment:
Is it possible to combine code to determine whether depthwise in True? For example, if len(weight_shape) == 4, then weight_shape is not None. And looks like the logic for group could be simplified too.

Reply:
Thanks for the review! I'll revise the change :)